src/templates/componentSampleTemplate.html

```diff
@@ -11,22 +11,11 @@
   <link rel="stylesheet" href="<%= componentName %>.css">
 
   <script type="text/javascript" src="https://code.jquery.com/jquery-2.1.4.min.js"></script>
- 
+  <script type="text/javascript" src="../../../../js/fabric.js"></script>
+  <script type="text/javascript" src="../../../../lib/pickadate.js"></script>
```

Comment:
This isn't needed by all of the components, but it's not hurting anything to be there. Especially if we're able to remove this plugin entirely at some point.